client/src/app/pages/controls/tags/components/tag-category-form.tsx

```diff
@@ -51,7 +50,7 @@ export const TagCategoryForm: React.FC<TagCategoryFormProps> = ({
   const { t } = useTranslation();
   const { pushNotification } = React.useContext(NotificationsContext);
 
-  const [error, setError] = useState<AxiosError>();
+  const [error] = useState<AxiosError>(); // TODO: Never set?
```

Comment:
It seems that all error handling is done in the callbacks via notifications. I would remove related code to avoid confusion.

Reply:
Done!